### PR TITLE
Add the Ruby language and its datatypes

### LINKED ISSUE #86: Add Ruby data types
# Description

The Ruby language is presently not on Code Thesaurus. I would love to see this added and the data types concept added to CT.

# Requirements

* Copy `web/thesauruses/_meta/data_types.json` to a `web/thesauruses/ruby/` directory
* Edit the data_types.json file to accurately reflect Ruby and the latest major version of it.

# Additional Notes

Please leave comments on this issue to ask questions or to claim the issue as yours. I'll keep it yours for a week then remove you in case someone else wants to work on it. (You can always reclaim it.)

Check out [CONTRIBUTING.md](CONTRIBUTING.md) for more information on how to contribute. 

Check out [https://docs.codethesaur.us/thesaurus/](https://docs.codethesaur.us/thesaurus/) for information on more specifics of implementing this.

### PR DESCRIPTION
**Pull request by sophiedeziel**
Closes #86

### CODE REVIEW

File: web/thesauruses/ruby/data_types.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    ],
+    "Complex Numbers": [
+      "complex_as_object",
+      "real_number_part",
+      "imaginary_number_part"
+    ]
+  },
+  "data_types": {
+    "boolean": {
+      "name": "",
```
Comment (geekygirlsarah) [inline web/thesauruses/ruby/data_types.json]: All the `name` fields can actually be removed from this file. They're not loaded into the site.

File: web/thesauruses/ruby/data_types.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+      "code": ""
+    },
+    "float_32_bit": {
+      "code": ""
+    },
+    "float_64_bit": {
+      "code": ""
+    },
+    "float_as_object": {
+      "name": "Object-based floating point",
```
Comment (geekygirlsarah) [inline web/thesauruses/ruby/data_types.json]: Same here, and lines 69 and 76.

File: web/thesauruses/ruby/data_types.json (hunk 1 above)
Comment (sophiedeziel) [reply web/thesauruses/ruby/data_types.json]: Thank you! I will push a new commit to fix that!

**Comment by geekygirlsarah** [comment]
Thank you for the first Ruby contribution! I'll gladly merge it in!

And hopefully there may be more Ruby contributions in the future! :)

**Pull request merged**